package.json

```diff
@@ -11,7 +11,9 @@
   "dependencies": {
     "array-flatten": "2.1.1",
     "debug": "2.6.9",
+    "depd": "^1.1.1",
     "methods": "~1.1.2",
+    "mwstack": "0.0.1",
```

Comment:
The repo of this dep probably needs cleaned up, especially with the wake of all the stuff people are looking at from the crypto incident.

Really, just the standard things that Express expects of it's deps

(1) there is no tag in the upstream repo corresponding to 0.0.1 version (or any branch where 0.0.1 is in package.json) and

(2) it needs to pin it's own deps so we're not pulling in a flexible dep into this module

I notice it doesn't work with all the same Node.js versions as this module does (when I looked at why CI is failing). Is that intentional? (just asking to confirm, not saying this is an issue).

Reply:
Personally I would like to move that repo under the express org if we are moving forward with this.  If that is the case I would propose we consider using the scope and also conforming that repo to all the normal standards we have.

I was in POC mode writing that, so I would be happy to whip it into shape before we merge here.

Reply:
I would this that particular module should be under `pillarjs` org, not the `expressjs` org. It also wouldn't fall under the `@express` scope proposal, as it is not an express module, but a pillar component (like the router itself). The router (a pillar component) should not have a dependency on something higher like an express part -- the tree goes the other way around.

Reply:
That all makes sense.  In my head I keep thinking of them more interchangeably, I will fix the way I phrase these comments in the future.

I am happy to move the repo into this github org if we are agreed we want to move forward with it.  If no one opposes this approach I will clean up the repo, add tests, move the repo and publish a `1.0.0` before we merge this.

Reply:
Hey @dougwilson, with the new npm scopes registered I can work on moving this over to this github org and published as `@pillarjs/mwstack`.  Can you give me the rights to publish new packages in the scope?